lib/vm/src/libcalls.rs

```diff
@@ -399,8 +398,53 @@ pub unsafe extern "C" fn wasmer_raise_trap(trap_code: TrapCode) -> ! {
 ///
 /// This function does not follow the standard function ABI, and is called as
 /// part of the function prologue.
-#[no_mangle]
-pub static wasmer_probestack: unsafe extern "C" fn() = PROBESTACK;
+cfg_if::cfg_if! {
+    if #[cfg(all(
+            target_os = "windows",
+            target_env = "msvc",
+            target_pointer_width = "64"
+            ))] {
```

Comment:
And if we want to ship this for nigthly-only, we should feature gate this code based on nightly vs not nightly